src/github.com/matrix-org/dendrite/clientapi/routing/routing.go

```diff
@@ -85,6 +105,14 @@ func Setup(
 		return writers.Register(req, accountDB, deviceDB)
 	}))
 
+	r0mux.Handle("/directory/room/{roomAlias}",
+		common.MakeAuthAPI("directory_room", deviceDB, func(req *http.Request, device *authtypes.Device) util.JSONResponse {
+			// TODO: Return the result of looking up the room in the room directory
```

Comment:
This is spurious surely? From this files point-of-view, it is calling the right thing with the right args.